python/Gaffer/__init__.py

```diff
@@ -61,3 +64,19 @@
 
 # Class-level non-UI metadata registration
 Metadata.registerValue( Reference, "childNodesAreReadOnly", True )
+
+def rootPath() :
+
+	return pathlib.Path( os.path.expandvars( "$GAFFER_ROOT" ) )
+
+# Returns the path of the Gaffer executable for the current platform.
+# If `resolved` is `True`, the full path will be returned, otherwise
```

Comment:
resolved -> resolve

Reply:
Fixed in 984f0088272e66f2a178b4cb62f7c92998d7db00